sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -434,13 +434,19 @@ def _get_availability_zones_with_error_handling(
                                regions)
     # Remove the regions that the user does not have access to.
     az_mappings = [m for m in az_mappings if m is not None]
+    errored_regions = collections.defaultdict(set)
+    for region, reason in errored_region_reasons:
+        errored_regions[reason].add(region)
     if errored_regions:
-        # This could happen if a AWS API glitch happens, it is to make sure
-        # that the availability zone does not get lost silently.
-        table = log_utils.create_table(['Regions', 'Reason'])
-        for reason, regions in errored_regions.items():
+        # This could happen if (1) an AWS API glitch happens, (2) permission
+        # error happens for specific availability zones. We print those zones to
+        # make sure that those zone does not get lost silently.
+        table = log_utils.create_table(['Region', 'Reason'])
```

Comment:
Oops, my mistake; didn't realize it's a list of regions (thought it's 1 row per region).
```suggestion
        table = log_utils.create_table(['Regions', 'Reason'])
```